### PR TITLE
Implemented Rail fence cipher exercise

### PR DESCRIPTION
**Pull request by kk-r**

### CODE REVIEW

**Comment by petemcfarlane** [comment]
Hi @kk-r there are no changed files here... did you mean to add some changes in your last commit?

**Comment by kk-r** [comment]
@petemcfarlane Sorry, Work In Progress, I didn't complete that exercise. Once done i'll update.

**Comment by kk-r** [comment]
@petemcfarlane Ready for review

File: exercises/rail-fence-cipher/rail-fence-cipher_test.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        $plainText  = "EXERCISES";
+        $rails = 4;
+        $expected = "ESXIEECSR";
+        $this->assertEquals(encode($plainText, $rails), $expected);
+    }
+
+    /**
+     * Test encode with two rails.
+     */
+    public function testDecodeWithTwoRails()
```
Comment (petemcfarlane) [inline exercises/rail-fence-cipher/rail-fence-cipher_test.php]: A minor change, but the test name and description say *two* rails, but the test itself uses *three* rails - wouldn't want to confuse people 😄

**Comment by kk-r** [comment]
@petemcfarlane  Done.

**Comment by petemcfarlane** [comment]
Great thanks!

**Pull request merged**